### DISCUSSION TITLE
The resolution of SNES games changes on certain scenes with all cores

### DISCUSSION THREAD

**Discussion by NikitaRUniverse** [OnionUI/Onion#614] [Q&A]
I found that in all Picross NP titles for SNES, during the tutorial and task selection, the screen resolution is distorted, but after the scene is completed, the resolution becomes normal. This happens with all cores on the console. Is there any way to fix this?

Comparasion of screen resolutions: 
https://sun9-82.userapi.com/impg/lHu-lybF1TyqA6raTG8A1FbHhHfu4COKPmAo3w/_GJUGirG98o.jpg?size=800x1600&quality=96&sign=a7bcc88ab86cc1eef7b3e774dd43f9a4&type=album
https://sun9-83.userapi.com/impg/93TrCeCpycKuPt9fmI1YLFhbAOZIP_IpgeeQew/AVawMpolJM8.jpg?size=800x1600&quality=96&sign=06770af1b18fcb38a515c85f975a5de7&type=album

**Comment by schmurtzm** [comment]
I can't reproduce, probably a setting in RA.... Try to reset your RA settings in Tweaks app (backup your settings first if you have specific RA configurations)
